web/src/datastore.js

```diff
@@ -115,6 +122,7 @@ export {
 	getProfile,
 	getMapFeatures,
 	getLocalizedText,
-	connectGndDatastore,
+  connectFeature,
```

Comment:
Nit: Indentation.

Reply:
Looking at the diff in GitHub the Indentation is still off. Please check your editor settings and if necessary format manually. Thanks!